web/react/components/more_direct_channels.jsx

```diff
@@ -47,6 +47,21 @@ export default class MoreDirectChannels extends React.Component {
         UserStore.addChangeListener(this.handleUserChange);
     }
 
+    componentDidUpdate(prevProps) {
+        if (!prevProps.show && this.props.show) {
+            this.onShow();
+        }
+    }
+
+    onShow() {
+        if ($(window).width() > 768) {
```

Comment:
There might be a way to do it with `max-height:calc(100% - 200px)`, but I think that requires a height to be set for every parent node if I remember correctly. I spent a small bit of time looking at it before, but I stopped because I had other stuff to work on. I'll file a ticket so we can actually budget some time for it